values-schema.yaml

```diff
@@ -64,14 +64,20 @@ definitions:
   annotations:
     $ref: '#/definitions/labelsAnnotations'
     description: A set of annotations.
-  azureMonitorWithoutCreds:
+  azureMonitor:
     properties:
       appInsightsApiKey:
         description: An Azure AppInsights client secret (defaults to clientSecret).
         type: string
       appInsightsAppId:
         description: An Azure AppInsights client id (defaults to clientId).
         type: string
+      clientId:
+        description: An Azure client id.
+        type: string
+      clientSecret:
+        description: An Azure client secret.
+        type: string
```

Comment:
where is `tenantId`? How did that get lost @j-zimnowoda (I am asking you first after the single-repo refactor)? This was coming from the `azure.tenantId` prop. Where did the root `azure` prop go? And how was the refactoring done of the templates that relied on those?